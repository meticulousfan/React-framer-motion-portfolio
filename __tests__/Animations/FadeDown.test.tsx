/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react";

import FadeDown from "../../src/components/Animations/FadeDown.component";

describe("FadeDown", () => {
  it("FadeDown laster inn og kan vises", () => {
    const { container } = render(<FadeDown delay={1}>FadeDown</FadeDown>);
    expect(container).toMatchSnapshot();
  });
});
