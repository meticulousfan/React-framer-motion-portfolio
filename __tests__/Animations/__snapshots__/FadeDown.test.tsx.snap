// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FadeDown FadeDown laster inn og kan vises 1`] = `
<div>
  <div
    style="opacity: 0; transform: translateY(-50px) translateZ(0);"
  >
    FadeDown
  </div>
</div>
`;
